docs/openapi/components/schemas/common/FSMAReceivingCTE.yml

```diff
@@ -0,0 +1,194 @@
+      '@id': https://schema.org/DateTime
+  additionalData:
+    title: Additional Data
+    description: Additional Key Data Elements (KDEs).
+    type: array
+    items:
+      $ref: ./FSMAAbstractKDE.yml
+    $linkedData:
+      term: additionalData
+      '@id': https://w3id.org/traceability#FSMAAbstractKDE
+additionalProperties: false
+example: |-
+  {
+    "type": "FSMAReceivingCTE",
+    "shipment": {
+      "type": "FSMAShipment",
+      "product": {
+        "type": "FSMAProduct",
+        "traceabilityLot": {
+          "type": "FSMATraceabilityLot",
+          "lotCode": "CHE-MIG-TTF1-061321-H37J",
+          "lotCodeAssignmentMethod": "Lot codes are assigned based on the commodity, variety, location, harvest date and a random 4 digit string.",
+          "lotCodeGeneratorLocation": {
+            "type": [
+              "Place"
+            ],
+            "address": {
+              "type": [
+                "PostalAddress"
+              ],
+              "organizationName": "John's Produce",
+              "streetAddress": "4335 Walsh Underpass",
+              "addressLocality": "Port Mark",
+              "addressRegion": "LA",
+              "postalCode": "67391",
+              "addressCountry": "United States"
+            },
+            "locationName": "John's Tomato Farm #1"
+          },
+          "lotCodeGeneratorPOC": {
+            "type": "Entity",
+            "entityType" : "Person",
+            "name" : "John Davis",
+            "email": "jdavis@johnsproduce.com",
+            "phoneNumber": "129.697.0673"
+          },
+          "lotType": "originated"
+        },
+        "quantityUnitMeasure": {
+          "type": [
+            "QuantitativeValue"
+          ],
+          "unitCode": "Field Bins",
+          "value": "103"
+        },
+       "additionalData": [
+          {
+            "type": "FSMAAbstractKDE",
+            "name": "Category code",
+            "value": "659351253"
+          },
+          {
+            "type": "FSMAAbstractKDE",
+            "name": "Category name",
+            "value": "Tomatoes"
+          },
+          {
+            "type": "FSMAAbstractKDE",
+            "name": "Brand name",
+            "value": "John's Produce"
+          },
+          {
+            "type": "FSMAAbstractKDE",
+            "name": "Commodity",
+            "value": "Cherry Tomatoes"
+          },
+          {
+            "type": "FSMAAbstractKDE",
+            "name": "Variety",
+            "value": "Mighty Sweet Hybrid"
+          },
+          {
+            "type": "FSMAAbstractKDE",
+            "name": "Product name",
+            "value": "N/A"
+          },
+          {
+            "type": "FSMAAbstractKDE",
+            "name": "Packaging size",
+            "value": "Bulk"
+          },
+          {
+            "type": "FSMAAbstractKDE",
+            "name": "Packaging style",
+            "value": "Field Bins"
+          }
+        ]
+      },
+      "from": {
```

Comment:
interesting semantics, why not `source` and `destination`?

Reply:
This is FDA language unfortunately

Reply:
Not very important but FWIW it's also the most clear language I can think of, or else I'd recommend shifting it 🤷‍♂️

`to` and `from` are properties on the FSMAShipment schema, which has a lot of data common to both `FSMAReceivingCTE` and `FSMAShippingCTE`. In both cases the `to` and `from` refer to the immediate previous destination and immediate subsequent destination of that leg of the journey.

The wording of the KDEs for both shipping & receiving is super clear about _immediate_ previous/subsequent locations, presumably so that the "source" for a product that's been shipped a lot is taken to mean the previous stop rather than the farm it was harvested on or something. So, whatever wording used has to imply this.

Given that `shipment` is used for KDEs shared between `shipping` and `receiving`, to/from seemed more clear than immediatePreviousSource/immediateNextDestination, or immediateSource/immediateDestination.